src/Input.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+function Input({ input, onChange, onClick }) {
```

Comment:
```suggestion
function Input({ input, onChange, onClickInsert }) {
```

handleClickInsert 와 쌍으로 지금처럼 맞춰주면 좋을 것 같네요.